xtask/src/main.rs

```diff
@@ -422,7 +422,12 @@ impl BuildTarget {
         match self {
             Self::OpteAdm => {
                 println!("Building opteadm ({p_name}).");
-                build_cargo_bin(&["--bin", "opteadm"], p_name, None, true)
+                build_cargo_bin(
+                    &["--bin", "opteadm"],
+                    p_name,
+                    Some("bin/opteadm"),
+                    true,
+                )
```

Comment:
Building from the workspace root appeared to be pulling in more dependencies via feature unification.

Reply:
Adding a comment in the code about this would be helpful.